From c1136bc4c2d1a6d9a3edb579f9dccfe529f552ed Mon Sep 17 00:00:00 2001
From: Chee Yang Lee <chee.yang.lee@intel.com>
Date: Fri, 11 Nov 2022 15:23:49 +0800
Subject: [PATCH] Makefile: skip imacros and test

do not need imacros and test the kernel module with bitbake.

also change gcc to bitbake gcc

Upstream-Status: Inappropriate [yocto specific]

Signed-off-by: Chee Yang Lee <chee.yang.lee@intel.com>
---
 qat_contig_mem/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 6cf7470..c0ef000 100644
--- a/Makefile
+++ b/Makefile
@@ -21,10 +21,10 @@ AUTO_CONF=$(KERNEL_SOURCE_ROOT)/$(PATH_AUTOCNF)/autoconf.h
 KDIR=$(KERNEL_SOURCE_ROOT)
 endif
 
-CC		:= gcc -Wall -imacros $(AUTO_CONF)
+CC		:= $(gcc) -Wall 
 
 ifeq ($(KERNELRELEASE),)
-all:	$(MODULENAME)_test
+#all:	$(MODULENAME)_test
 all:
 	$(MAKE) -C $(KDIR) M=$(shell pwd) modules
 else
-- 
2.37.3

